css/selectors/contains-invalid.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Selectors: selector list containing invalid selector</title>
+    <link rel="help" href="https://drafts.csswg.org/selectors-4/#grouping">
+    <meta name="assert" content="If a selector is invalid, the entire selector list is invalid.">
+    <meta name="assert" content="Pseudo-elements are not valid within :is :not :where.">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <style>
+      .a, .b, .c {
+        color: lime;
+      }
+
+      .a, :nonsense {
+        color: red;
+      }
+
+      /* Pseudo-elements cannot be represented by the matches-any pseudo-class; they are not valid within :is(). */
+      .b, :is(::before) {
+        color: red;
+      }
+      .b, :not(::before) {
+        color: red;
+      }
+      .b, :where(::before) {
+        color: red;
+      }
+
+      .c:is(*, :nonsense) {
+        color: red;
+      }
+      .c:not(*, :nonsense) {
```

Comment:
There's remaining discussion in https://github.com/w3c/csswg-drafts/issues/3264#issuecomment-440752606 about what should happen to `:not()` with invalid arguments. As currently resolved, it should match (so the test would be wrong), but maybe we'll change that, so I would wait for the dust to settle before landing this PR, as this may or may not need fixing.